allennlp/training/trainer.py

```diff
@@ -621,7 +621,7 @@ def _validation_loss(self) -> Tuple[float, int]:
 
         return val_loss, batch_num
 
-    def train(self) -> None:
+    def train(self) -> dict:
```

Comment:
it looks like you could use `Dict[str, float]`?

Reply:
Having these be printed/logged as a side effect of them being returned seems nasty...